requirements-test.in

```
@@ -0,0 +1 @@
+pytest
```

Kudos for using pip-tools! Small suggestion, it's probably a good idea to use constraints:
https://jamescooke.info/a-successful-pip-tools-workflow-for-managing-python-package-requirements.html
https://github.com/jazzband/pip-tools

```suggestion
-c requirements.txt
pytest
```